plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -42,6 +42,7 @@
 {
     public static final String FILE_FORMAT_PROPERTY = "format";
     public static final String PARTITIONING_PROPERTY = "partitioning";
+    public static final String SORTED_BY_PROPERTY = "sorted_by";
```

Comment:
`SHOW CREAT TABLE` result doesn't have this property. Please fix and add a test to cover it.